pkg/operators/v1alpha1/subscription_types.go

```diff
@@ -118,8 +118,18 @@ const (
 	// SubscriptionBundleUnpackFailed indicates that the unpack job failed
 	SubscriptionBundleUnpackFailed SubscriptionConditionType = "BundleUnpackFailed"
 
-	// SubscriptionOperatorDeprecated indicates that the Operator currently installed with this Subscription has been deprecated.
```

Comment:
I suspect this condition is used on olm side. So this is kind of an api change. Is it necessary to remove this without deprecation warning?

Reply:
For context, we just added that field yesterday and it's not in any release, so it's not in use anywhere. We're making a correction to the name here and also adding sub-types so that users can see which levels their installed Operators are deprecated at.